src/docs/asciidoc/developer-manual.adoc

```diff
@@ -170,110 +170,7 @@ For now, environments are not fully supported but this feature will be available
 
 
 === Create your account
-
-Currently, the only way to get a ZetaPush account is by contacting us. The aim is to understand your needs
-and to advise you even if you just want to test it:
-
-* link:https://www.zetapush.com/sign-up-for-a-free-trial/[Signup for a free trial]
-* link:https://www.zetapush.com/contact/[Contact us for more suitable account]
-
-Your account will be created soon after the first contact and an application that fits your
-needs will be available. Then you are ready to start working with ZetaPush.
-
-[NOTE]
-====
-Soon we will open access to everyone for free trial. Account creation will be available through
-our CLI tool, from the main site and the web console.
-====
-
-
-////
-When you use ZetaPush the first time, you need to create an account. Once your account is ready, you
```

Comment:
I agree to remove comments but I think we should put this in a new file that we can include later.
Once we allow users to signup themselves, we will have to rewrite this again. Instead, just save it in another file and we will change the include